charts/backstage/templates/backstage-deployment.yaml

```diff
@@ -62,6 +62,9 @@ spec:
           {{- range .Values.backstage.args }}
             - {{ . | quote }}
           {{- end }}
+          {{- if .Values.backstage.resources }}
+          resources: {{- toYaml .Values.backstage.resources | nindent 12 }}
```

Comment:
I found myself not understanding bitnami common enough, but I think @sabre1041 may suggest to use `common.tplvalues.render` here instead.

https://github.com/backstage/charts/pull/20#pullrequestreview-1214201433

Reply:
Should be able to get more information [here](https://artifacthub.io/packages/helm/bitnami/common#tplvalues) and [here](https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_tplvalues.tpl).

Reply:
Yeah, I know that, I was just contemplating the overall benefit of using it here. :smile:

Reply:
Sounds like kinda have a consensus we want to be able to use the functionality `common.tplvalues.render` provides as much as we can - correct? The only drawback seems to be the syntax (and therefore ease of the use for newcommers) compared to `toYaml`, correct?

I think @sabre1041's preference is summed here, lol https://github.com/backstage/charts/pull/29 

I can see the benefit of both approaches, maybe we can just write it down into some contributions guideline, something like a note for people to consider replacing `toYaml` with `common.tplvalues.render` + some examples to clarify the syntax? Would that at least mitigate the problem?

Reply:
I think if as a team there's a reason given and we help folks understand how to contribute/use the common.tplvalues.render it will get to a middle ground there. I would say @sabre1041 is the one with the strong opinion and the rest of us seem to be a "eh whichever way" and in these cases the strong opinion typically wins. I'll make the changes today to use the tplvalues.render. Will leave the documention update to others.

Given the strong preference for the tplvalues.render from @sabre1041. I would say they are best situated to do said documentation effectively.

Reply:
updated with the tplvalues.render.